README.md

```diff
@@ -18,6 +18,25 @@ If any changes are made to Protobuf message definitions in the specification.md
 sure to update also the `*.proto` files in the [proto](proto) directory. Run `make gen-proto`
 to run the Protobuf compiler and verify the `*.proto` files.
 
+## Proposal Requirements
+
+Proposals to add new capabilities to OpAMP specification must be accompanied by working
+prototypes in [opamp-go](https://github.com/open-telemetry/opamp-go), demonstrating
+the capability. The prototype can be a PR against opamp-go or can be on your own fork
+of opamp-go.
```

Comment:
```suggestion
of opamp-go. The proposal will be used to help aid us in understanding the implications of the specification changes and how actual usage might look.
```
I think a motivation for this requirement would help contributors understand what is being looked for in a prototype.

Reply:
Done.